packages/editor/src/components/rich-text/style.scss

```diff
@@ -38,7 +38,7 @@
 		color: $dark-gray-800;
 		background: $light-gray-200;
 		font-family: $editor-html-font;
-		font-size: 14px;
+		font-size: inherit; // This is necessary to override upstream CSS.
```

Comment:
How does front-end look?

Reply:
Good question.

This is editor:

![screen shot 2018-08-13 at 14 43 59](https://user-images.githubusercontent.com/1204802/44032493-8cb913f2-9f07-11e8-80f9-5b9ff3f4cc98.png)

This is TwentySomething theme:

![screen shot 2018-08-13 at 14 44 32](https://user-images.githubusercontent.com/1204802/44032504-9389ced8-9f07-11e8-8c79-307d82c94e3d.png)

☝️ note how the code is given a blanket font size there.

<a href="https://github.com/jasmussen/navi">This is the guten theme I've been playing with</a>:

![screen shot 2018-08-13 at 14 45 25](https://user-images.githubusercontent.com/1204802/44032537-ad213174-9f07-11e8-8de8-18e9803482dd.png)